lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -253,6 +261,11 @@ defmodule ExUnit.Case do
 
     quote bind_quoted: binding do
       test = :"test #{message}"
+
+      if Module.defines?(__MODULE__, {test, 1}) do
+        raise ExUnit.DuplicateTestError, message: ~s(A test named "#{message}" is already defined in #{__MODULE__}.)
```

Comment:
Explanation:

  * messages start with lower case
  * use inspect for better module name format
  * no need for :message key